depends/install_imagequant.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # install libimagequant
 
-archive=pngquant-2.6.0
+archive=pngquant-2.8.2
 
```

Comment:
This didn't actually work due to a change in the project/makefile:
https://travis-ci.org/python-pillow/Pillow/jobs/187587442#L2240